website/src/releases/2023-05/new-theme.mdx

```diff
@@ -0,0 +1,19 @@
+---
+date: 2023-05-09
+title: New UI style for the Merchant Center
+description: The UI style of the Merchant Center is now updated, following its redesign.
+type: announcement
+topics:
+  - Merchant Center
+  - Development
+---
+
+Now that we've redesigned the Merchant Center to enhance user experience and usability, you can also make your Custom Application look consistent with the rest of the Merchant Center.
+You'll just need to update your `ui-kit` and `app-kit` dependencies (use the latest version if possible).
+
+We don't expect you to need to do any extra work regarding the styles of your components.
+
+Additionally, as part of the new Merchant Center design, we introduced new [layout guidelines for displaying content on a page](/development/page-content-layouts), including new UI components to implement these layouts.
+We recommend for Custom Applications to start following these guidelines and use the page content layout components.
+
+As always, if you have questions or feedback you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```

Comment:
@CarlosCortizasCT I've made minimal edits keeping in mind the priority of the message. Ok now?
```suggestion
You can now get your Custom Application to resemble the default applications in the redesigned Merchant Center. To achieve this, update the `ui-kit` and `app-kit` dependencies to the latest version. We don't expect you to do any extra work for the style of your components.

The new Merchant Center design also brings new [layout guidelines](/development/page-content-layouts) to display page content, and new UI components to implement the layouts. We recommend following these guidelines and using the layout components for your Custom Applications.

If you have questions or feedback, open a GitHub [discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or [issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```